src/routes/safe/components/Apps/confirmTransactions.tsx

```diff
@@ -41,21 +42,43 @@ const IconText = styled.div`
   }
 `
 const isTxValid = (t) => {
```

Comment:
can we add return type here?

Reply:
what's going inside this function? what difficulties did you experience with types that made you do those hacky things?

Reply:
that's to ensure the TX coming from a sapp has the correct values. for `to` `value` and `data`

Reply:
> can we add return type here?
It's inferred.

Reply:
yes, but we have an eslint rule for explicit return types: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-function-return-type.md

Reply:
> that's to ensure the TX coming from a sapp has the correct values. for to value and data

there's type `unknown` for this case